roles/calibre-web/defaults/main.yml

```diff
@@ -0,0 +1,26 @@
+# The values here are defaults.
+# To override them edit /etc/iiab/local_vars.yml
+
+# Installation Variables
+calibreweb_install: False
+calibreweb_enabled: False
+
+# calibre-web folder to store its data and configuration files.
+calibreweb_home: "{{ content_base }}/calibre-web"
+
+calibreweb_port: 8083
+calibreweb_url: /calibre-web
+calibreweb_path: "{{ iiab_base }}/calibre-web"
+calibreweb_exec_path: "{{ calibreweb_path }}/cps.py"
+
+# Calibre-web setup will be provisioned with default administration account, metadata.db and
+# language. You could turn this to 'False' while reinstalling/upgrading calibre-web.
+calibreweb_provision: True
+calibreweb_database: metadata.db
+
+#calibre-web system user
+calibreweb_user: root
+
+# calibreweb admin account
+# calibreweb_admin_user: admin
+# calibreweb_admin_password: changeme
```

Comment:
Recommendation: change calibreweb_admin_user to `Admin` with uppercase A.

(And if possible uncomment both above lines?!)

Reply:
@holta These variables are not used. Currently the provisioning is done using the SQLite database 'metadata.db'. There is no code associated/using these variables.